### PR TITLE
Feature[C4R-842] Search Organizations with practice programs

### PR DESCRIPTION
**Pull request by birloiflorian**
- [C4R-842](https://wearetribus.atlassian.net/browse/C4R-842) Search organizations with active practice programs

### CODE REVIEW

File: backend/src/modules/organization/constants/errors.constants.ts [hunk 1]
```diff
@@ -91,6 +91,10 @@ export const ORGANIZATION_ERRORS = {
     message: 'Erorr while updateing organization Completion Status',
     errorCode: 'ORG_021',
   },
+  GET_ORGANIZATIONS_WITH_ACTIVE_PRACTICE_PROGRAMS: {
+    message: 'Erorr while loading organizations with practice programs',
```
Comment (dragos1195) [inline backend/src/modules/organization/constants/errors.constants.ts:95]: Typo

File: backend/src/modules/organization/services/organization.service.ts [hunk 2]
```diff
@@ -331,6 +335,89 @@ export class OrganizationService {
...
+        };
+        organizationActivity?: {
+          domains: {
+            id: FindOperator<number>;
+          };
+        };
+      } = {
+        ...filters,
+        practicePrograms: {
+          active: true, // get only organizations with active practice programs
```
Comment (dragos1195) [inline backend/src/modules/organization/services/organization.service.ts:362]: Also you should add a filter for Active organizations.

**Review by dragos1195 — CHANGES_REQUESTED** [review]

File: backend/src/modules/organization/constants/errors.constants.ts (hunk 1 above)
Comment (birloiflorian) [reply backend/src/modules/organization/constants/errors.constants.ts:95]: Fixed

File: backend/src/modules/organization/services/organization.service.ts (hunk 2 above)
Comment (birloiflorian) [reply backend/src/modules/organization/services/organization.service.ts:365]: Done

**Pull request merged**